### DISCUSSION TITLE
Car Position

### DISCUSSION THREAD

**Discussion by np2446** [kutu/pyirsdk#42] [General]
Hello, is there a way to get the x, y, and z values of where on the track the car is? I don't see anything in vars.txt but not sure if I'm missing something.

**Comment by kutu** [comment]
live - no
ibt - yes, but only your car

**Reply by np2446** [reply]
I'm looking to build an overlay that shows you which sector of the track you are in (mainly focused on learning nords sector names but will build out other tracks too). However seems pretty impossible without live data. If you are familiar with iOverlay, do you know how they are able to build their overlay that shows everyone's position on the track (I think they might actually be open source so I'll look into that)

**Reply by np2446** [reply]
Actually figured it out, I dont know how I missed the var that shows the pct of the track you've completed.
